flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/rules/logical/CorrelateSortToRankRuleTest.scala

```diff
@@ -95,34 +94,35 @@ class CorrelateSortToRankRuleTest extends TableTestBase {
   }
 
   @Test
-  def testAggCallNotSupported(): Unit = {
+  def testNonInnerJoinNotSupported(): Unit = {
```

Comment:
It seems that these tests are just out of order. Reorder the tests and clean up unnecessary changes?

Reply:
+1

Reply:
please add some tests about equal with constant value, such as  `WHERE f0 = t2.f0 AND f1 = 1` or `WHERE f0 = t2.f0 AND 1 = t2.f1`